src/ripple/nodestore/Database.h

```diff
@@ -27,6 +27,7 @@
 #include <ripple/nodestore/impl/Tuning.h>
 #include <ripple/nodestore/Scheduler.h>
 #include <ripple/nodestore/NodeObject.h>
+#include <ripple/protocol/digest.h>
```

Comment:
Woops. I forgot about the builds I had going to check if this `#include` is necessary. It's not. I'm going to leave my approval in place, since removing it is a small correction.

Reply:
Will remove, thanks.

Reply:
Ah, the header is required if RIPPLE_VERIFY_NODEOBJECT_KEYS is defined. I'll warp the include as follows.

#if RIPPLE_VERIFY_NODEOBJECT_KEYS
#include <ripple/protocol/digest.h>
#endif